Gruntfile.js

```diff
@@ -322,13 +333,10 @@ module.exports = function(grunt) {
     }
   });
 
-  // global beforeEach task
```

Comment:
OK, although I found it slightly confusing due to its similarity to testing terminology - like it was something special. How about I modify it to something more humane?